include/rocksdb/utilities/transaction_db.h

```diff
@@ -101,6 +101,11 @@ struct TransactionDBOptions {
   // ordering rather than concurrency control.
   bool skip_concurrency_control = false;
 
+  // This option is only valid for write unprepared. If a write batch exceeds
+  // this threshold, then the transaction will implicitly flush the currently
+  // pending writes into the database. A value of 0 or less means no limit.
+  ssize_t default_write_batch_flush_threshold = 0;
```

Comment:
ssize_t is used rarely in the code base. What adv it brings over size_t in this case?

Reply:
This is because `TransactionOptions::write_batch_flush_threshold` is signed. A value of -1 there means "unspecified".

If `default_write_batch_flush_threshold` were unsigned, then we would have to be more careful when assigning `default_write_batch_flush_threshold` onto `default_write_batch_flush_threshold`, because the max of size_t is larger than ssize_t.

I guess I could add validation logic in `WriteUnpreparedTxnDB::Initialize` to check that `default_write_batch_flush_threshold` is less than max of ssize_t.